### ISSUE TITLE
highlighter needs refresh

### ISSUE THREAD

**Issue by jmikulo** [EastArctica/JS-Plugins#9]
quizizz beta highlighter needs page refresh every question, its annoying. btw i uses highlighter script via extension

**Comment by rraanan** [comment]
the highlighter is really annoying and also I would really appreciate it if the answers option could be put back on, it was really helpful.

**Comment by EastArctica** [comment]
Hmmm, We're looking into this issue. Does the quiz have the leaderboard option turned off?

**Comment by jmikulo** [comment]
> Hmmm, We're looking into this issue. Does the quiz have the leaderboard option turned off?

yes - leaderboard off, assigned, can not re-attempt, can not view answers when finished

**Comment by Brayanv891** [comment]
el resaltador RC1 tiene problemas a veces subraya la respuesta y algunas veces no era para saber si lo pueden arreglar 
gracias por su atencion
